flowr/examples/running-average/DESCRIPTION.md

```
@@ -0,0 +1,44 @@
+# Running Average
+
+This example implements the running average calculation from the Lucid dataflow
+programming language book ("Lucid, the Dataflow Programming Language" by Wadge
+and Ashcroft, 1985, page 47).
+
+## The Lucid Program
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)